### ISSUE TITLE
Change Image Functionality

### ISSUE THREAD

**Issue by orangepowered** [ResetCollection/support#135]
Instead of downloading all images or a small amount (1-5), can we instead just download 1 for everything and then if we want to update an image you provide an image chooser to let us select the new one?  On a similar note, can you provide an option to allow us to remove all images that aren't currently being used?

I am imagining something similar to how poweramp works.

This would massively speed up the time of your application to scrape images I would imagine.

**Comment by dnichols4** [comment]
There's already a setting for "Max number of downloads per art type".  Does that not achieve what you're looking for?

**Comment by orangepowered** [comment]
Not at all.  If I don't like an image I only have the 4 ut found to choose
from.  The amount space the app uses is still large, it would make far more
sense if I could download just 1 of each and then if one is wrong I
could then do a lookup for for the top X number of items and choose the
best one.  That would set the icon to the new image and delete the old one,
saving a massive amount of space.

**Comment by dover19** [comment]
I agree, the scraping functionality is very frustrating to use. Basically you need to set the amount of downloads as high as you can, then go through your games one by one to set the correct box art. Even after you've done all of that, there's no way to remove the junk you don't actually want.

It would also help if there was a way to use our own art without having to manually select it in the UI. For example, if I use a 3rd party app to scrape my art, I should be able to simply use a naming convention to associate my art to the correct rom files and jsut upload them all to the proper images folder.

Another way to improve the scraper would be to offer more precise scraping options. Let me decide if I prefer 3d boxart (when available) and use the flat front cover as a fallback. Also, let me decide if I want US, EU or JP covers. Right now it's all over the place and there's no simple way to replace all the boxart I don't want with the correct.

**Comment by microton79** [comment]
A function to delete all unused Media would be appreciated :)
